File: inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/messages/request/SaveWordAlignmentRequest.java
package de.tudarmstadt.ukp.inception.experimental.api.messages.request;

public class SaveWordAlignmentRequest
{
    private String clientName;
    private String userName;
    private long projectId;
    private int sentence;
    private String alignments;

    public String getClientName()
    {
        return clientName;
    }

    public void setClientName(String aClientName)
    {
        clientName = aClientName;
    }

    public String getUserName()
    {
        return userName;
    }

    public void setUserName(String aUserName)
    {
        userName = aUserName;
    }

    public long getProjectId()
    {
        return projectId;
    }

    public void setProjectId(long aProjectId)
    {
        projectId = aProjectId;
    }

    public int getSentence()
    {
        return sentence;
    }

    public void setSentence(int aSentence)
    {
        sentence = aSentence;
    }

    public String getAlignments()
    {
        return alignments;
    }

    public void setAlignments(String aAlignments)
    {
        alignments = aAlignments;
    }
}

File: inception/inception-api-annotation-experimental/src/main/ts/client/messages/request/SaveWordAlignmentRequest.ts
export class SaveWordAlignmentRequest
{
    clientName: string;
    userName: string;
    projectId: number;
    sentence: number;
    alignments: string;

    constructor(aClientName: string, aUserName: string, aProjectId: number, aSentence: number, aAlignments: string)
    {
        this.clientName = aClientName;
        this.userName = aUserName;
        this.projectId = aProjectId;
        this.sentence = aSentence;
        this.alignments = aAlignments;
    }
}